website/docs/developer/document-store.md

```diff
@@ -0,0 +1,138 @@
+---
+id: document-store
+title: Document Store
+sidebar_label: Document Store
+---
+
+The [Document Store](https://github.com/Open-Attestation/document-store) repository contains both the smart contract code for document store (in `/contracts`) as well as the node package for using this library (in `/src`).
+
+## Installation
+
+```sh
+npm i @govtechsg/document-store
+```
+
+---
+
+## Usage
+
+To use the package, you will need to provide your own Web3 [provider](https://docs.ethers.io/ethers.js/html/api-providers.html) or [signer](https://docs.ethers.io/ethers.js/html/api-wallet.html) (if you are writing to the blockchain).
+
+#### Table of Contents
+
+- [Deploy](#deploy)
+- [Connect](#connect-to-existing-document-store)
+- [Deploy (with minimal proxy)](#deploy-with-minimal-proxy)
+- [Interact](#interact)
+- [List of available functions](#list-of-available-functions)
+- [Provider & Signer](#provider--signer)
+- [Setup](#setup)
+- [Contract Benchmark](#contract-benchmark)
+- [Notes](#notes)
+
+### Deploy
+
```

Comment:
copy the titles? `Deploying new document store` ? thought they were more descriptive. same with the rest

Reply:
ok done.